ci-pr.nix

```diff
@@ -28,11 +28,17 @@ let
         do
           # ignore all errors
           echo -n $file
-          if timeout 10s -- moc $file -no-check-ir -ref-system-api -o $file.wasm 2>/dev/null
+          if timeout 10s moc $file -no-check-ir -ref-system-api -o $file.wasm 2>/dev/null
           then echo " failed (ignored)"
           else echo " ok"
           fi
         done
+
+        if ! find -name \*.wasm | grep -q .
```

Comment:
```suggestion
        if ! find -name */*.wasm | grep -q .
```

Reply:
(not CI actually fails due to broken pipe around here...

Reply:
That changes the meaning. `\*.wasm` passes a pattern to `find`, `*/*.wasm` uses the shell to expand.

Reply:
Actually, I couldn't figure out how to fix it myself. Any suggestions?

Reply:
I think #2644 fixes this...

Reply:
Oh, there was breakage on CI? I thought you are just suggesting a different implementation.

Are you sure the broken pipe causes the error?

Anyways, won't be able to look until at the laptop again. (Phone right now)